### ISSUE TITLE
MISSING_SESSION warning not triggered on ds000247

### ISSUE THREAD

**Issue by psadil** [bids-standard/bids-validator#432]
The current specification (1.2.4) defines a "MISSING_SESSION" error code: https://github.com/bids-standard/bids-schema/blob/1fbd48df925d60686a191838e1affd5cb8df1b7f/versions/master/schema/rules/errors.yaml#L149-L153

From what I'm seeing, that would need a custom implementation that's not currently in the bids-validator. For example, bids-examples/ds000247 does not trigger it.

```shell
# fits structure that I'd expect being warned about, given that "Not all subjects contain the same sessions."
❯ ls bids-examples/ds000247/sub-*
bids-examples/ds000247/sub-0002:
ses-0001

bids-examples/ds000247/sub-0003:
ses-0001

bids-examples/ds000247/sub-0004:
ses-0001

bids-examples/ds000247/sub-0006:
ses-0001

bids-examples/ds000247/sub-0007:
ses-0001

bids-examples/ds000247/sub-emptyroom:
ses-18901014	ses-18901015	ses-18910228	ses-18910303	ses-18910512

# config for this run...
❯ cat bids-examples-config.json
{
    "ignore": [
        {
            "code": "EMPTY_FILE"
        },
        {
            "code": "NIFTI_HEADER_UNREADABLE"
        },
        {
            "code": "GZ_NOT_GZIPPED"
        },
        {
            "code": "NIFTI_TOO_SMALL"
        }
    ]
}

❯ deno run -ERWN jsr:@bids/validator --version
bids-validator 2.4.1

# no message about MISSING_SESSION ?
❯ deno run -ERWN jsr:@bids/validator bids-examples/ds000247 --config bids-examples-config.json --ignoreNiftiHeaders | grep -i SESSION
          6 - Subjects 6 - Sessions        rest                    MRI                  
```

**Comment by effigies** [comment]
Agreed that it's missing. Looking back at our [issue triage](https://docs.google.com/spreadsheets/d/1o1JMk3w9Ge9YtgetnwQWE14_jt8mdnN7OfbVJxneX2I/edit?gid=0#gid=0), we marked this as WONTFIX.

I'm curious what value people see in this one? I think ds000247 is a great example of false positives, as sub-emptyroom is not supposed to look like other subjects.

**Comment by psadil** [comment]
FWIW, I'd be fine with dropping the rule. It reads to me as though it's meant to catch accidental bid-conversion errors more than to suggest a way to improve the quality of a dataset.
